### ISSUE TITLE
Searching "Elixir" in VS Code Marketplace doesn't show Lexical as a result

### ISSUE THREAD

**Issue by Nezteb** [lexical-lsp/vscode-lexical#54]
Despite having "elixir" as an extension tag, when [searching the VS Code extension marketplace for "elixir"](https://marketplace.visualstudio.com/search?term=elixir&target=VSCode&category=All%20categories&sortBy=Relevance
), Lexical shows up very low on the list:
<img width="1193" alt="screenshot_2023-10-19_at_19 31 22@2x" src="https://github.com/lexical-lsp/vscode-lexical/assets/3588798/4b1fd431-2493-493e-befe-26529cd6cd08">

When searching my installed extensions for "elixir", Lexical doesn't show up at all:
<img width="492" alt="screenshot_2023-10-19_at_19 29 47@2x" src="https://github.com/lexical-lsp/vscode-lexical/assets/3588798/5e9792df-ac66-481e-9873-610c93d4284a">

I'm _guessing_ this is because the search favors extensions that actually have "elixir" in the name? The only thing I can think to try is to rename the extension `Lexical (Elixir Language Server)` or something.

**Comment by Blond11516** [comment]
Hi @Nezteb! Thanks for raising the issue, that's definitely something we should try to address.

I've just merged #55 which updates the extenion's display name as you suggested. I'll try to publish a new version of the extension soon with the change.

Once I've verified that the change has the desired result I'll close the issue. If it doesn't I'll keep the issue open to research other ways of improving our position in the search.

**Comment by Blond11516** [comment]
New name is now published. Looks like we've jumped from ~50th position to ~25th. Not quite as high as I'd hoped, but still pretty good!

Our position now seems consistent with our number of downloads compared to other extensions around the same position, so I guess that makes sense.

I also saw you added a positive review @Nezteb, thanks a lot for that! It's always nice to get those and I'm sure it helps with the search placement as well.

I'll close this issue. I don't know that there's a lot more we can do on this apart from growing Lexical's community.
